src/controller.ts

```diff
@@ -495,7 +495,7 @@ export function Head(url: string): MethodDecorator {
  * @returns {MethodDecorator} - A MethodDecorator tailored to denote that the adorned method should
  * handle incoming CONNECT requests for the provided URL path.
  */
-export function CONNECT(url: string): MethodDecorator {
+export function Connect(url: string): MethodDecorator {
```

Comment:
Why do you only use capital letters for `Connect`?😲

Reply:
it was a just typo. 😅

Reply:
Ah, I misunderstood that it was changed from uppercase to lowercase.